DQMServices/Components/plugins/EDMtoMEConverter.h

```diff
@@ -56,15 +56,16 @@ class EDMtoMEConverter : public edm::EDAnalyzer
   ~EDMtoMEConverter() override;
   void beginJob() override;
   void endJob() override;  
-  void analyze(const edm::Event&, const edm::EventSetup&) override;
   void beginRun(const edm::Run&, const edm::EventSetup&) override;
   void endRun(const edm::Run&, const edm::EventSetup&) override;
   void beginLuminosityBlock(const edm::LuminosityBlock&, const edm::EventSetup&) override;
   void endLuminosityBlock(const edm::LuminosityBlock&, const edm::EventSetup&) override;
-  void respondToOpenInputFile(const edm::FileBlock&) override;
+  void respondToOpenInputFile(const edm::FileBlock&) ;
```

Comment:
if this is never called, you could simply drop it, as well as `iCountf`

Reply:
will be removed.